src/Raven.Client/Documents/Operations/Replication/PutPullReplicationAsHubOperation.cs

```diff
@@ -24,6 +32,9 @@ public PutPullReplicationAsHubOperation(string name)
             _pullReplicationDefinition = new PullReplicationDefinition(name);
         }
 
+        /// <inheritdoc cref="PutPullReplicationAsHubOperation"/>
+        /// <param name="pullReplicationDefinition">The pull replication hub definition to apply.</param>
+        /// <exception cref="ArgumentException">Thrown if the <see cref="PullReplicationDefinition.Name"/> is null or empty.</exception>
```

Comment:
need to document `PullReplicationDefinition`

Reply:
done